src/components/ListingDetails/styles.tsx

```diff
@@ -0,0 +1,83 @@
+import styled from 'styled-components';
+import { H1, P } from '@/styles/text';
+import COLORS from '@/styles/colors';
+
+export const CaseDisplay = styled.aside`
+  position: relative;
+  width: 100%;
+`;
+
+export const CaseTitle = styled(H1)`
+  margin: 0.4rem 4rem;
+`;
+
+// export const CaseInterestContainer = styled.div`
```

Comment:
let's go ahead and remove these commented lines if we won't need them moving forward